pkg/webhook/resources/virtualmachine/validator.go

```diff
@@ -92,6 +220,10 @@ func (v *vmValidator) Update(_ *types.Request, oldObj runtime.Object, newObj run
 		return nil
 	}
 
+	if err := v.checkForDuplicateMacAddrs(newVM); err != nil {
```

Comment:
the check of duplicated MAC is a heavy operation, and Harvester other controllers may update vm object other fileds like https://github.com/harvester/harvester/pull/7195 is doing, could you add an additional condition, only when VM's interface / MAC address are updating/adding than the oldVM, then call this check, thanks.

Reply:
Thanks, done